### PR TITLE
feat: lavender theme refactor

### LINKED ISSUE #3047: [Bug] lavenedar theme same as default in sidebar?
### Bug Description

as described in the title, the sidebar isn't consistent with the lavender theme
image

### How to Reproduce

image

### Expected Behavior

Like the dandelion theme, the sidebar should be under the lavender theme

### Operating System

Windows 11

### AppFlowy Version(s)

v0.2.7

### Screenshots

_No response_

### Additional Context

_No response_

### LINKED ISSUE #3062: [Bug] Wrong selected color in SettingsDialog
### Bug Description

The background color is not correct after clicking it(if cursor didn't move after it)

### How to Reproduce

Go the the settings dialog, click one option(do not move cursor) the background becomes the light blue.

https://github.com/AppFlowy-IO/AppFlowy/assets/14248245/ea760496-7012-44fe-934c-a49e33f66bde

### Expected Behavior

After making a selection, the main color should appear as normal blue without any light blue visible on the screen. 

### Operating System

macOS

### AppFlowy Version(s)

main

### Screenshots

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by hyj1204**
### Feature Preview
1. Improve the lavender theme color.
 Fix #3047 
 Light mode
Pasted Graphic 7
Pasted Graphic 8
Dark mode
image

image

3. Fix #3062 

4. Reduce the scrollbar size from 12 to 8
Pasted Graphic 6

---

#### PR Checklist

- [x] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [x] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/packages/flowy_infra_ui/lib/style_widget/hover.dart [hunk 1]
```diff
@@ -13,6 +11,10 @@ class FlowyHover extends StatefulWidget {
   final void Function(bool)? onHover;
   final MouseCursor? cursor;
 
+  /// Reset the hover state when the parent widget get rebuild.
+  /// Default to true.
+  final bool? resetHoverOnRebuild;
```
Comment (a-wallen) [inline frontend/appflowy_flutter/packages/flowy_infra_ui/lib/style_widget/hover.dart:16]: If it defaults to true, does this type need to be nullable?

**Review by a-wallen — APPROVED** [review]

File: frontend/appflowy_flutter/packages/flowy_infra_ui/lib/style_widget/hover.dart [hunk 2]
```diff
@@ -29,6 +31,7 @@ class FlowyHover extends StatefulWidget {
     this.isSelected,
     this.onHover,
     this.cursor,
+    this.resetHoverOnRebuild = true,
     this.buildWhenOnHover,
   }) : super(key: key);
```
Comment (a-wallen) [inline frontend/appflowy_flutter/packages/flowy_infra_ui/lib/style_widget/hover.dart:36]:
```suggestion
    bool? resetHoverOnRebuild = true,
    this.buildWhenOnHover,
  }) : resetHoverOnRebuild = resetHoverOnRebuild, super(key: key);
```

Comment (a-wallen) [reply frontend/appflowy_flutter/packages/flowy_infra_ui/lib/style_widget/hover.dart:36]: Something like this if it's already defaulting to true.

File: frontend/appflowy_flutter/packages/flowy_infra_ui/lib/style_widget/hover.dart (hunk 1 above)
Comment (hyj1204) [reply frontend/appflowy_flutter/packages/flowy_infra_ui/lib/style_widget/hover.dart]: You are right, I removed the question mark.

**Review by MayurSMahajan — CHANGES_REQUESTED** [review]
Can we change the sidebar's background color in **dark mode** to: 
`sidebarBg: const Color(0xff2D223B)` to make it more purple-dominant? 
Also, we should change the main2 color which affects the hover color to be either equal to the same color as main1 color OR some shade close to main1 like: 
`main2: const Color(0xffB00FFF)`

**Comment by AmanNegi** [comment]
Lavendar in light mode looks so pleasant <3

**Pull request merged**